Kickstarter-iOS/Views/Cells/LiveStreamChatMessageCell.swift

```diff
@@ -0,0 +1,80 @@
+import Library
+import LiveStream
+import Prelude
+
+internal final class LiveStreamChatMessageCell: UITableViewCell, ValueCell {
+  private let viewModel: LiveStreamChatMessageCellViewModelType = LiveStreamChatMessageCellViewModel()
+
+  @IBOutlet private weak var avatarImageView: UIImageView!
+  @IBOutlet private weak var avatarImageViewWidthConstraint: NSLayoutConstraint!
+  @IBOutlet private weak var contentStackView: UIStackView!
+  @IBOutlet private weak var creatorIndicatorDotImageView: UIImageView!
+  @IBOutlet private weak var creatorIndicatorDotImageViewWidthConstraint: NSLayoutConstraint!
+  @IBOutlet private weak var creatorTextLabel: UILabel!
+  @IBOutlet private weak var nameLabel: UILabel!
+  @IBOutlet private weak var nameStackView: UIStackView!
+  @IBOutlet private weak var messageLabel: UILabel!
+  @IBOutlet private weak var rootStackView: UIStackView!
+
+  internal func configureWith(value chatMessage: LiveStreamChatMessage) {
+    self.viewModel.inputs.configureWith(chatMessage: chatMessage)
+  }
+
+  internal required init?(coder aDecoder: NSCoder) {
+    super.init(coder: aDecoder)
+
+    self.selectionStyle = .none
+    self.transform = CGAffineTransform(scaleX: 1, y: -1)
+  }
+
+  internal override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.contentView
+      |> UIView.lens.backgroundColor .~ .ksr_navy_700
+
+    _  = self.avatarImageView
+      |> UIImageView.lens.layer.masksToBounds .~ true
+      |> UIImageView.lens.backgroundColor .~ .ksr_grey_500
+
+    _ = self.rootStackView
+      |> UIStackView.lens.alignment .~ .top
+      |> UIStackView.lens.spacing .~ Styles.grid(1)
+      |> UIStackView.lens.layoutMargins .~ .init(topBottom: CGFloat(Styles.grid(2)) / 2,
+                                                 leftRight: Styles.grid(2))
+      |> UIStackView.lens.layoutMarginsRelativeArrangement .~ true
+
+    _ = self.creatorTextLabel
+      |> UILabel.lens.text .~ localizedString(key: "Creator", defaultValue: "Creator")
```

Comment:
Opened yesterday, still waiting to confirm if we need all the strings before adding them: https://github.com/kickstarter/kickstarter/pull/6207